Classes/BEMAnalogClockView.m

```diff
@@ -320,12 +337,28 @@ - (void)handlePan:(UIPanGestureRecognizer *)recognizer {
     } else {
         if (self.hours >= 24) {
             self.hours = 00;
-        } else if (self.hours <= 0) {
+        } else if (self.hours < 0) {
             self.hours = 23;
         }
     }
+    
+    if (self.touchHourHand == YES) {
+        if (self.hours >= 12 && self.hours <= 24) {
+            self.hours =  ((((atan2f((translation.x - self.frame.size.height/2), (translation.y - self.frame.size.width/2)) * -(180/M_PI) + 180))/6)/5) + 12;
```

Comment:
@matheusalano, I believe that @werediver is referring to the fact that both calculations are very similar. The only difference seems to add 12 if the condition is met.

You could do something like that:
```objective-c
CGFloat hours = (((atan2f((translation.x - self.frame.size.height/2), (translation.y - self.frame.size.width/2)) * -(180/M_PI) + 180))/6)/5;
if (self.hours >= 12 && self.hours <= 24) {
    hours += 12;
}
self.hours = hours;
```
*I haven't tested this code by the way*.